pkg/asset/installconfig/gcp/validation.go

```diff
@@ -133,3 +133,18 @@ func validateMachineNetworksContainIP(fldPath *field.Path, networks []types.Mach
 	}
 	return field.ErrorList{field.Invalid(fldPath, subnetName, fmt.Sprintf("subnet CIDR range start %s is outside of the specified machine networks", ip))}
 }
+
+func CheckForServicesToBeEnabled(projectServices []string) error {
```

Comment:
```suggestion
func ValidateEnabledServices(services []string) error {
```
remember in Go, the package name is usually part of the call site, so keeping that in mind `gcp.ValidateEnabledServices` is much closer describing what the function does.